### PR TITLE
253 multi laon by company yields unweighted prod as ald

### PR DESCRIPTION
**Pull request by maurolepore**
Closes #253

Unweighted production by company should ignore loan_size_* columns.

### CODE REVIEW

**Comment by georgeharris2deg** [comment]
Wonderful - Thanks @maurolepore 

Looks good to me - Will await @jdhoffato have the final say. 
 
I will notify all the relevant Users once we can push this to CRAN (unfortunately most banks can't use devtools as Gitub is often blocked on their systems. 

Thanks

**Comment by maurolepore** [comment]
Thanks! Beware that CRAN folks are on vacation until Jan 4.

**Review by jdhoffa — APPROVED** [review]
Yup this looks good to me! 
@georgeharris2deg this should now properly output the unweighted production.

**Pull request merged**
